currency-map.js

```diff
@@ -362,5 +362,337 @@ module.exports = {
     "TWD": {
         "name": "Taiwan Dollar (TWD)",
         "symbolFormat": "TWD {#}"
+    },
+    "CZK": {
+        "name": "Czech Koruna",
+        "symbolFormat": "CZK {#}"
+    },
+    "FIM": {
+        "name": "",

```

Comment:
Just starting looking up some of these with a blank name and they were replaced with the EURO. Should we remove these from the list? I don't think we'd want users to ever select these right? @ahaug1289 how would we handle these from a data perspective?